main.js

```diff
@@ -1,6 +1,100 @@
 // wait for DOM to load before running JS
 $(function() {
   
-  // your code here
+	var turnCount = 0;
+	// var winArray = [[0,1,2],[0,3,6],[2,5,8],[6,7,8],[0,4,8],[2,4,6],[1,4,7],[3,4,5]];
+
+	// $('.box').on('click', function() {
+	// 	turnCount++;
+	// 	if (turnCount % 2 === 0) {
+	// 		$(this).html('<p>X</p>');
+	// 		$(this).off('click');
+	// 	}else {
+	// 		$(this).html('<p>O</p>');
+	// 		$(this).off('click');
+	// 	}
+	// })
+
+	function play (event) {
+		$('.box').on('click', function() {
+			turnCount++;
+			if(turnCount % 2 === 0) {
+				$(this).addClass('x')
+				$(this).html('<p>X</p>');
+				$(this).off('click');
+			}else {
+				$(this).addClass('o')
+				$(this).html('<p>O</p>');
+				$(this).off('click')
+			}
+		})
+	}
+
+  $('#reset').on('click',function(){
+    $('.box').empty();
+    $('.box').html('<p>+</p>')
+    play();
+  })
+
+play();
+/**
+  function getWinner(xORo) {
+		for (i = 0; < winArray.length; i++) {
+			var possibleWins = winArray[i];
+			var spotCount = 0;
+			for (j = 0 < xORo.length; j++ ) {
+				if (possibleWins.indexOf(xORo[j]) > -1) {
+					spotCount++;
+				}
+				if (spotCount === 3) {
+					break;
+				}
+			}
+  }
+**/
+
+
+
+//get winner
+//very dirty way 
+var x = 'x';
+var o = 'o';
+var spot1 = $('#a');
+var spot2 = $('#b');
+var spot3 = $('#c');
+var spot4 = $('#d');
+var spot5 = $('#e');
+var spot6 = $('#f');
+var spot7 = $('#g');
+var spot8 = $('#h');
+var spot9 = $('#i');
+
+$('.box').on('click', function() {
+	if(spot1.hasClass('o') && spot2.hasClass('o') && spot3.hasClass('o') ||
```

Comment:
I can't tell but I think without parenthesis this won't work but it does work on the console. So I can't say it is wrong but I'm shocked it works. I would have expected:

```js
( spot1.hasClass('o') && spot2.hasClass('o') && spot3.hasClass('o') ) ||
(spot4.hasClass('o') && spot5.hasClass('o') && spot6.hasClass('o'))
```